components/eamxx/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -129,7 +129,7 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Computed>("SW_clrsky_flux_dn_dir", scalar3d_layout_int, Wm2, grid_name, ps);
   add_field<Computed>("LW_clrsky_flux_up", scalar3d_layout_int, Wm2, grid_name, ps);
   add_field<Computed>("LW_clrsky_flux_dn", scalar3d_layout_int, Wm2, grid_name, ps);
-  add_field<Computed>("rad_heating_pdel", scalar3d_layout_mid, Pa*K/s, grid_name, ps);
+  add_field<Updated>("rad_heating_pdel", scalar3d_layout_mid, Pa*K/s, grid_name, ps);
```

Comment:
That seems to have done it. I removed the xml/yaml mods and just added these fields to the RESTART list, reverting them all to Computed rather than Updated.